File: csharp/1512.number-of-good-pairs.cs
/*
 * @lc app=leetcode id=1512 lang=csharp
 *
 * [1512] Number of Good Pairs
 */

// @lc code=start
public class Solution {
    public int NumIdenticalPairs(int[] nums) {
        int ret = 0;
        var count = new int[101];
        foreach (var x in nums) {
            ret += count[x]++;
        }
        return ret;
    }
}
// @lc code=end

File: java/1512.number-of-good-pairs.java
/*
 * @lc app=leetcode id=1512 lang=java
 *
 * [1512] Number of Good Pairs
 */

// @lc code=start
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int[] count = new int[101];
        int ret = 0;
        for (int n : nums) {
            ret += count[n]++;
        }
        return ret;
    }
}
// @lc code=end

